src/lib/content.d.ts

```diff
@@ -293,6 +293,12 @@ interface NumberedTitleBlockElement {
 	format: CAPIFormat;
 }
 
+interface InteractiveContentBlockElement {
+	_type: 'model.dotcomrendering.pageElements.InteractiveContentBlockElement';
```

Comment:
Real nitpick but I would think of it as a `Contents` block element rather than `Content`. Maybe there's a better name that isn't so easily confused with the word "content" though.